tests/integration_tests.rs

```diff
@@ -70,6 +70,26 @@ fn run_single_test_success() {
         .success();
```

Comment:
@fmoko One little detail that I noticed: the `verify_all_failure` test case misled me a bit, since it only tests that _some_ exercise fails to verify. Should we change the name?